inference-engine/tests/functional/shared_test_classes/src/read_ir/generate_inputs.cpp

```diff
@@ -0,0 +1,97 @@
+                                             node->get_mode());
+        return blob;
+    }
+    return FuncTestUtils::createAndFillBlob(info.getTensorDesc());
+}
+
+InferenceEngine::Blob::Ptr generate(const std::shared_ptr<ngraph::op::v0::ROIPooling> node,
+                                    const InferenceEngine::InputInfo& info) {
+    const auto& inputShape = info.getInputData()->getDims();
+    if (inputShape.size() == 2) {
+        InferenceEngine::Blob::Ptr blob;
+        blob = make_blob_with_precision(info.getTensorDesc());
+        blob->allocate();
+
+        CommonTestUtils::fill_data_roi(blob->buffer(),
+                                       blob->size(),
+                                       node->get_input_shape(0).front() - 1,
+                                       inputShape[2],
+                                       inputShape[3],
+                                       1.0f,
+                                       node->get_method() == "max");
+        return blob;
+    }
+    return FuncTestUtils::createAndFillBlob(info.getTensorDesc());
+}
+
+InferenceEngine::Blob::Ptr generate(const std::shared_ptr<ngraph::op::v3::ROIAlign> node,
+                                    const InferenceEngine::InputInfo& info) {
+    const auto& inputShape = info.getInputData()->getDims();
+    switch (inputShape.size()) {
+        case 1: {
+            std::vector<int> roiIdxVector(node->get_shape()[0]);
+            ROIAlignLayerTest::fillIdxTensor(roiIdxVector, node->get_shape()[0]);
+            return FuncTestUtils::createAndFillBlobWithFloatArray(info.getTensorDesc(), roiIdxVector.data(), roiIdxVector.size());
+        }
+        case 2: {
+            std::vector<float> blobData(node->get_shape()[0] * 4);
+            ROIAlignLayerTest::fillCoordTensor(blobData,
+                                               inputShape[2],
+                                               inputShape[3],
+                                               node->get_spatial_scale(),
+                                               node->get_sampling_ratio(),
+                                               node->get_pooled_h(),
+                                               node->get_pooled_w());
+            return FuncTestUtils::createAndFillBlobWithFloatArray(info.getTensorDesc(), blobData.data(), blobData.size());
+        }
+        default:
+            return FuncTestUtils::createAndFillBlob(info.getTensorDesc());
+    }
+}
+
+template<typename T>
+InferenceEngine::Blob::Ptr generateInput(const std::shared_ptr<ngraph::Node> node,
+                                         const InferenceEngine::InputInfo& info) {
+    return generate(ngraph::as_type_ptr<T>(node), info);
+}
+} // namespace
+
+InputsMap getInputMap() {
+    static InputsMap inputsMap{
+#define NGRAPH_OP(NAME, NAMESPACE) {NAMESPACE::NAME::type_info, generateInput<NAMESPACE::NAME>},
+    #include <shared_test_classes/read_ir/opset_int_tbl.hpp>
```

Comment:
why are you using upset table from tests? Why not to use ngraph_opset#_tbl?

Reply:
fixed